frontend/src/components/TextArea.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import App from "../app";
+
+const TextArea = ({link, ...rest}) => (
+  <textarea
+    style={ {"overflowY": "scroll", "height": "150px"}}
+    onChange=
+      { (e) => { App.save(link, e.currentTarget.value); } }
```

Comment:
Note that this was `App.save("list", e.currentTarget.value);` which worked with the code as is since the only instance of `TextArea` had `link` set to `"list"` but would be broken if re-used in other places.